src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -438,10 +438,10 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>The lock file for {0} at location {1} does not exist, no-op is not possible. Continuing restore.</value>
   </data>
   <data name="Warning_MinVersionDoesNotExist" xml:space="preserve">
-    <value>{0} depends on {1} but {2} was not found. An approximate best match of {3} was resolved.</value>
```

Comment:
I think you want the strings.designer.cs regenerated as well, otherwise, someone else will get these changes in their future PR.

Reply:
+1